### DISCUSSION TITLE
Missing Legacy Skin Elements (Song Progress & Key Overlay)

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#17496] [Ideas] [answered]
Quick post to address some missing legacy skin elements.
Most skins these days have some form of skin element inside the healthbar to skin the song progress so either bar or pie mode in stable but the option to add the legacy song progress bar or pie is missing from the skin editor making that element look out of place / incomplete. I have also heard multiple complaints about the new song progress element being very distracting and I personally agree to an extent. I feel we should be able to add this back into our skins.

This also ties into the legacy key overlay being missing as a LOT of skins also include a design for this in the healthbar. There is no skin element for adding back this legacy key overlay to complete our skins intended design on lazer.

**Comment by Theighlin** [comment, marked as answer]
#10944
#10945

Progress graph can be disabled in the settings.

**Reply by bdach** [reply]
and that progress graph setting may be deleted too because now you can just delete the element via skin editor.
